test/templates/list-group/component.njk

```diff
@@ -1,23 +1,12 @@
 {% import "../forms.njk" as forms %}
-{% from "../button/component.njk" import button %}
 {% from "../icon/component.njk" import icon %}
```

Comment:
We shouldn't be importing the icon component.

Reply:
I think I removed it

Reply:
@droshev once you remove this import statement I think this is good to 🚢 !